.changeset/integration.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vanilla-extract/integration': major
+---
+
+Use absolute paths internally to make sure Vite resolves modules correctly
```

Comment:
Worth noting that this change will only affect integrations that use the new compiler, which is currently only esbuild-next and vite.